docs/react/list-other.mdx

```diff
@@ -2,44 +2,30 @@
 sidebar_position: 4
 ---
 
-# Display other peers media
+# Display other peers' media
```

Comment:
Are you sure about `peers'`? 🤔

Reply:
it's correct, but we could also go with
`Display media of other peers`